src/Compilers/Test/Utilities/CSharp/TestOptions.cs

```diff
@@ -32,5 +34,10 @@ public static class TestOptions
 
         public static readonly CSharpCompilationOptions UnsafeDebugDll = DebugDll.WithAllowUnsafe(true);
         public static readonly CSharpCompilationOptions UnsafeDebugExe = DebugExe.WithAllowUnsafe(true);
+
+        public static CSharpCompilationOptions WithStrictMode(this CSharpCompilationOptions options)
+        {
+            return options.WithFeatures(ImmutableArray.Create("strict"));
```

Comment:
Is this additive? I would expect WithStrictMode helper to preserve whatever features already set.